CART/CARTLib/utils/data.py

```diff
@@ -60,6 +61,56 @@ def load_segmentation(path: Path):
     return segment_node
 
 
+def load_markups(path: Path) -> list[slicer.vtkMRMLMarkupsFiducialNode]:
+    """
+    Load a file into Slicer as a Markups node.
+
+    Unlike slicer's default utility function, it will hide the markups from view
+    by default to better work with CART's iterative DataUnit loading.
+
+    Also there is a workaround to track all loaded markup nodes
+    :param path: Path to the file
+    """
+    # Load the file into a markups node, hidden from view
+
+    all_fiducials = set(slicer.util.getNodesByClass("vtkMRMLMarkupsFiducialNode"))
+
+    markups_nodes = slicer.util.loadMarkups(
+        path
+    )  # THIS IS SUPPOSED TO RETURN A LIST if
+    # there are multiple nodes in the file
+    # THIS DOESNT https://slicer.readthedocs.io/en/latest/developer_guide/slicer.html#slicer.util.loadMarkups
+    all_new_fiducials = list(
+        set(slicer.util.getNodesByClass("vtkMRMLMarkupsFiducialNode")) - all_fiducials
+    )
+
+    print(f"Found {len(all_new_fiducials)} new markups nodes after loading {path}")
+    print(f"Fiductrials in the scene: {[node.GetName() for node in all_new_fiducials]}")
```

Comment:
Do we still need these print statments? If not, remove them. If so, theres a typo:

```suggestion
    print(f"Found {len(all_new_fiducials)} new markups nodes after loading {path}")
    print(f"Fiducials in the scene: {[node.GetName() for node in all_new_fiducials]}")
```